projects/isy-angular-widgets-demo/src/app/feature/dashboard/dashboard.component.spec.ts

```diff
@@ -61,33 +62,33 @@ describe('DashboardComponent', () => {
     expect(component).toBeTruthy();
   });
 
-  it('should have been initialized correctly', () => {
-    expect(component).toBeTruthy();
-  });
-
-  it('should have 4 chart data rows', () => {
-    expect(component.chartData.length).toEqual(4);
-  });
+  describe('with available charts', ()=> {
+    it('should init char data correctly', () => {
+      component.initCharts();
+      expectChartDataBeInit();
+    });
 
-  it('should have the german language as default language', () => {
-    expect(component.translate.langs.length).toEqual(1);
-    expect(component.translate.langs[0]).toEqual(GERMAN_LANGUAGE);
-    expect(component.translate.currentLang).toEqual(GERMAN_LANGUAGE);
-  });
+    it('should have correctly initialized chart data', () => {
```

Comment:
why do we need this test, `should init char data correctly` seems already to check everything from this test

Reply:
This test is checking the initialization of the dashboard widgets data

Reply:
Deleted